python/pyhail/dataset.py

```diff
@@ -313,12 +497,62 @@ def annotate_samples_vds(self, right, root=None, code=None):
     def annotate_variants_bed(self, input, root, all=False):
         """Annotate variants with a .bed file.
 
+        **Examples**
+
```

Comment:
Same comment as with `annotate_samples_table` about simpler examples at the beginning with one line description.